web-animations-1/Overview.bs

```diff
@@ -4809,10 +4809,10 @@ interface KeyframeEffect : AnimationEffect {
 :   <dfn attribute for=KeyframeEffect>pseudoElement</dfn>
 ::  The [=target pseudo-selector=].
     This is `null` if the [=effect target=] is an {{Element}} or is itself `null`.
-    When not `null`, this must be a valid [=pseudo-element=] selector,
-    attempting to set it to an invalid value will have no effect.
-    and will always return using the the double-colon syntax
-    (i.e. always `::before` and never `:before`).
+    When not `null`, this must be a [=pseudo-element=] selector with valid 
+    [[selectors-4#pseudo-element-syntax]].
```

Comment:
Just curious, how does this actually render? Does it say, "...a pseudo element selector with valid syntax"?

Reply:
oops, I accidentally had it include the section number, fixing that now.

Reply:
Now it renders like `a pseudo element selector with [=valid syntax=]`